### PR TITLE
EVA-1713 EVA-1714 - Update Help and API sections of the website

### PR DESCRIPTION
**Pull request by sundarvenkata-EBI**
EVA_Variant_Level_Normalisation.svg - moved to optimized section. Diagram source is noted in the **data-svg-source** attribute of the **img** tag.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -359,9 +359,18 @@ <h3>Some example of queries include:</h3>
                             <p>* To fetch all the variants in a segment region:
                                 <br/> <a href="http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37" target="_blank">http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37</a></p>
 
-                            <p><br/> * To fetch all the info of a variant:
+                            <p><br/> * To fetch all the info for a variant by ID:
```
Comment (jmmut) [inline src/index.html]: The PR is fine right now, although I suggest changing the first line that says 
```
The general structure of a EVA REST web service URL is:
http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}
```
to
```
The general structure of a EVA REST web service URL is one of:
http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}
http://www.ebi.ac.uk/eva/webservices/identifiers/{version}/{category}/ID
```
or something similar

File: src/index.html [hunk 2]
```diff
@@ -359,9 +359,18 @@ <h3>Some example of queries include:</h3>
                             <p>* To fetch all the variants in a segment region:
                                 <br/> <a href="http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37" target="_blank">http://www.ebi.ac.uk/eva/webservices/rest/v1/segments/11:128446-128446/variants?species=hsapiens_grch37</a></p>
 
-                            <p><br/> * To fetch all the info of a variant:
+                            <p><br/> * To fetch all the info for a variant by ID:
                                 <br/> <a href="http://www.ebi.ac.uk/eva/webservices/rest/v1/variants/rs666/info?species=hsapiens_grch37" target="_blank">http://www.ebi.ac.uk/eva/webservices/rest/v1/variants/rs666/info?species=hsapiens_grch37</a></p>
 
+                            <p><br/> * To fetch locus info for a variant by <a href="https://www.ncbi.nlm.nih.gov/books/NBK44374/#Content.what_is_the_ncbi_assay_id_or__ss" target="_blank">SS ID</a>:
```
Comment (andresfsilva) [inline src/index.html]: I noticed in the preview that some users are not very familiar with the terms RS IDs and SS IDs but they are familiar with RefSnp and SubSnp so we could maybe mention it here too as we do in the swagger of the identifiers

File: src/index.html [hunk 3]
```diff
@@ -343,25 +343,38 @@ <h2>Feedback</h2>
                     </nav>
                     <h2>API</h2>
                     <div>
-                        <h3>The general structure of a EVA REST web service URL is:</h3><p><a href="#">http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}</a></p>
+                        <h3>The general structure of a EVA REST web service URL is one of:</h3>
+                        <ul>
+                            <li><p><a href="#">Queries to retrieve variant information (locus, genotypes, evidence etc.,) - http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}</a></p></li>
```
Comment (jmmut) [inline src/index.html]:
```suggestion
                            <li><p><a href="https://www.ebi.ac.uk/eva/webservices/rest/swagger-ui.html">Queries to retrieve variant information (locus, genotypes, evidence etc.,) - http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}</a></p></li>
```

File: src/index.html [hunk 4]
```diff
@@ -343,25 +343,38 @@ <h2>Feedback</h2>
                     </nav>
                     <h2>API</h2>
                     <div>
-                        <h3>The general structure of a EVA REST web service URL is:</h3><p><a href="#">http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}</a></p>
+                        <h3>The general structure of a EVA REST web service URL is one of:</h3>
+                        <ul>
+                            <li><p><a href="#">Queries to retrieve variant information (locus, genotypes, evidence etc.,) - http://www.ebi.ac.uk/eva/webservices/rest/{version}/{category}/IDs/{resource}?{filters}</a></p></li>
+                            <li><p><a href="#">Queries to retrieve locus information by accession ID - http://www.ebi.ac.uk/eva/webservices/identifiers/{version}/{category}/ID[/submitted]</a></p></li>
```
Comment (jmmut) [inline src/index.html]:
```suggestion
                            <li><p><a href="https://www.ebi.ac.uk/eva/webservices/identifiers/swagger-ui.html">Queries to retrieve locus information by accession ID - http://www.ebi.ac.uk/eva/webservices/identifiers/{version}/{category}/ID[/submitted]</a></p></li>
```

**Pull request merged**